website/content/docs/commands/setup/vault.mdx

```diff
@@ -0,0 +1,153 @@
+---
+layout: docs
+page_title: 'Commands: setup vault'
+description: |
+  Setup a Vault cluster for Nomad integration.
+---
+
+# Command: setup vault
+
+This command sets up Vault for allowing Nomad workloads to authenticate
+themselves using Workload Identity.
+
+This command requires `acl:write` permissions for Vault and respects
+`VAULT_TOKEN`, `VAULT_ADDR`, and other [Vault-related environment
+variables][vaultenv].
+
+<Warning>
+
+This command is an experimental feature and may change its behavior in future
+versions of Nomad.
+
+</Warning>
+
+## Usage
+
+```plaintext
+nomad setup vault [options]
+```
+
+## General Options
+
+@include 'general_options_no_namespace.mdx'
```

Comment:
```suggestion
## General Options

@include 'general_options_no_namespace.mdx'
```

Same reason as for `setup consul`. I don't think we make any Nomad API calls in this command.